### ISSUE TITLE
Carousel style 2 change the scroll location when playing

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/wet-boew#6533]

**Comment by pjackson28** [comment]
Probably because a setfocus event is being fired on every tab rotation. Maybe the tabbed interface should be updated to not setfocus when a play-related tab rotation occurs (so only exception to the setfocus approach).

**Comment by pjackson28** [comment]
Will be fixed by PR #6573.
